wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/rest/referendumvorlage/ReferendumvorlageController.java

```diff
@@ -0,0 +1,47 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.rest.referendumvorlage;
+
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.services.referendumvorlage.ReferendumvorlageService;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.rest.model.WlsExceptionDTO;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/businessActions/referendumvorlagen")
+@RequiredArgsConstructor
+public class ReferendumvorlageController {
```

Comment:
```suggestion
public class ReferendumvorlagenController {
```
und Package Name im Plural? falls dies nicht bewusst so bezeichnet wurde.

Reply:
auf Plural wechseln

Reply:
fixed mit 35a0661b65da81ab7b55415e98bf308f422ae503